spec/features/user_can_delete_posts_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.feature "Timeline", type: :feature do
+  scenario "Can delete posts" do
+    post = Post.create(message: "Hello, world!")
+    visit "/posts/#{post.id}"
+    click_link "Delete"
+    expect(page).to_not have_content("Hello, world!")
+  end
```

Comment:
You could be testing for the amount of posts visible on the page
`posts = page.all('.post')`
`expect(posts.length).to eq SOME_NUMBER`